{
  try {
    ff.function(functionName,ff.literal(13));
  }
 catch (  IllegalArgumentException e) {
    return;
  }
  fail("IllegalArgumentException not thrown");
}
